TODO-ucr-config-files.md
git grep -l '@!@'

base/univention-app-appliance/conffiles/usr/share/univention-web/js/umc/hooks/appliance.json
base/univention-base-files/conffiles/etc/apt/apt.conf.d/20secureapt
base/univention-base-files/conffiles/etc/apt/apt.conf.d/80proxy
base/univention-base-files/conffiles/etc/cron.d/univention-config-registry-backup
base/univention-base-files/conffiles/etc/cron.d/univention-ucr-cronjobs
base/univention-base-files/conffiles/etc/default/locale
base/univention-base-files/conffiles/etc/default/ntpdate
base/univention-base-files/conffiles/etc/e2fsck.conf
base/univention-base-files/conffiles/etc/hosts.d/00-base
base/univention-base-files/conffiles/etc/hosts.d/20-static
base/univention-base-files/conffiles/etc/issue
base/univention-base-files/conffiles/etc/issue.net
base/univention-base-files/conffiles/etc/kernel-img.conf
base/univention-base-files/conffiles/etc/locale.gen
base/univention-base-files/conffiles/etc/logrotate.d/rsyslog
base/univention-base-files/conffiles/etc/logrotate.d/univention-config-registry-replog
base/univention-base-files/conffiles/etc/lsb-release
base/univention-base-files/conffiles/etc/modprobe.d/univention-blacklist.conf
base/univention-base-files/conffiles/etc/modules
base/univention-base-files/conffiles/etc/network/interfaces.d/10-default
base/univention-base-files/conffiles/etc/ntp.conf
base/univention-base-files/conffiles/etc/profile
base/univention-base-files/conffiles/etc/resolv.conf
base/univention-base-files/conffiles/etc/rsyslog.conf
base/univention-base-files/conffiles/etc/security/packetfilter.d/20_rsyslog.sh
base/univention-base-files/conffiles/etc/ssh/ssh_config
base/univention-base-files/conffiles/etc/ssh/sshd_config
base/univention-base-files/conffiles/etc/sysctl.d/local.conf
base/univention-base-files/conffiles/etc/systemd/journald.conf
base/univention-base-files/conffiles/etc/welcome.msg
base/univention-bootsplash/conffiles/etc/systemd/system/getty@.service.d/nottyreset.conf
base/univention-firewall/conffiles/etc/security/packetfilter.d/10_univention-firewall_start.sh
base/univention-firewall/conffiles/etc/security/packetfilter.d/20_docker.sh
base/univention-firewall/conffiles/etc/security/packetfilter.d/80_univention-firewall_policy.sh
base/univention-grub/conffiles/etc/default/grub
base/univention-heimdal/conffiles/etc/default/heimdal-kdc
base/univention-heimdal/conffiles/etc/heimdal-kdc/kadmind.acl
base/univention-heimdal/conffiles/etc/heimdal-kdc/kdc.conf
base/univention-heimdal/conffiles/etc/krb5.conf
base/univention-heimdal/conffiles/etc/logrotate.d/heimdal-kdc
base/univention-initrd/conffiles/etc/initramfs-tools/initramfs.conf
base/univention-maintenance/conffiles/etc/cron.d/univention-mrtg
base/univention-maintenance/conffiles/etc/cron.d/univention-system-stats
base/univention-maintenance/conffiles/etc/logrotate.d/univention-maintenance
base/univention-network-manager/conffiles/etc/default/ifplugd
base/univention-network-manager/conffiles/etc/dhcp/dhclient.conf
base/univention-pam/conffiles/etc/cron.d/univention-pam
base/univention-pam/conffiles/etc/libnss-ldap.conf
base/univention-pam/conffiles/etc/nscd.conf
base/univention-pam/conffiles/etc/nsswitch.conf
base/univention-pam/conffiles/etc/pam.d/chfn
base/univention-pam/conffiles/etc/pam.d/chsh
base/univention-pam/conffiles/etc/pam.d/common-account
base/univention-pam/conffiles/etc/pam.d/common-auth-nowrite
base/univention-pam/conffiles/etc/pam.d/common-auth.d/30univention-pam_local
base/univention-pam/conffiles/etc/pam.d/common-auth.d/50univention-pam_general
base/univention-pam/conffiles/etc/pam.d/common-password
base/univention-pam/conffiles/etc/pam.d/common-session.d/10univention-pam_common
base/univention-pam/conffiles/etc/pam.d/cron
base/univention-pam/conffiles/etc/pam.d/ftp
base/univention-pam/conffiles/etc/pam.d/gdm
base/univention-pam/conffiles/etc/pam.d/kcheckpass
base/univention-pam/conffiles/etc/pam.d/kde
base/univention-pam/conffiles/etc/pam.d/kdm
base/univention-pam/conffiles/etc/pam.d/kscreensaver
base/univention-pam/conffiles/etc/pam.d/login
base/univention-pam/conffiles/etc/pam.d/other
base/univention-pam/conffiles/etc/pam.d/passwd
base/univention-pam/conffiles/etc/pam.d/ppp
base/univention-pam/conffiles/etc/pam.d/rlogin
base/univention-pam/conffiles/etc/pam.d/rsh
base/univention-pam/conffiles/etc/pam.d/screen
base/univention-pam/conffiles/etc/pam.d/sshd
base/univention-pam/conffiles/etc/pam.d/su
base/univention-pam/conffiles/etc/pam.d/sudo
base/univention-pam/conffiles/etc/pam_ldap.conf
base/univention-pam/conffiles/etc/security/access-chfn.conf
base/univention-pam/conffiles/etc/security/access-chsh.conf
base/univention-pam/conffiles/etc/security/access-cron.conf
base/univention-pam/conffiles/etc/security/access-ftp.conf
base/univention-pam/conffiles/etc/security/access-gdm.conf
base/univention-pam/conffiles/etc/security/access-kcheckpass.conf
base/univention-pam/conffiles/etc/security/access-kde.conf
base/univention-pam/conffiles/etc/security/access-kdm.conf
base/univention-pam/conffiles/etc/security/access-kscreensaver.conf
base/univention-pam/conffiles/etc/security/access-login.conf
base/univention-pam/conffiles/etc/security/access-other.conf
base/univention-pam/conffiles/etc/security/access-passwd.conf
base/univention-pam/conffiles/etc/security/access-ppp.conf
base/univention-pam/conffiles/etc/security/access-rlogin.conf
base/univention-pam/conffiles/etc/security/access-rsh.conf
base/univention-pam/conffiles/etc/security/access-screen.conf
base/univention-pam/conffiles/etc/security/access-sshd.conf
base/univention-pam/conffiles/etc/security/access-su.conf
base/univention-pam/conffiles/etc/security/access-sudo.conf
base/univention-pam/conffiles/etc/security/limits.conf
base/univention-pam/conffiles/etc/security/pam_env.conf
base/univention-passwd-store/conffiles/etc/pam.d/common-auth.d/40univention-passwd-store
base/univention-passwd-store/conffiles/etc/pam.d/common-session.d/90univention-passwd-store
base/univention-printclient/conffiles/etc/cups/client.conf
base/univention-quota/conffiles/etc/pam.d/common-session.d/20univention-quota_common
base/univention-quota/conffiles/etc/samba/smb.conf.d/81univention-quota_scripts
base/univention-server/conffiles/etc/logrotate.d/univention-server-password-change
base/univention-server/reregister_ox_ldap_schema.sh
base/univention-ssl/conffiles/etc/logrotate.d/univention-ssl
base/univention-sudo/conffiles/etc/sudoers.d/univention
base/univention-system-activation/conffiles/usr/share/univention-system-activation/entries.json
base/univention-system-activation/conffiles/usr/share/univention-system-activation/languages.json
base/univention-system-setup/conffiles/etc/apache2/conf-available/univention-system-setup.conf
base/univention-system-setup/conffiles/etc/logrotate.d/univention-system-setup
base/univention-system-setup/conffiles/etc/motd.setup
base/univention-system-setup/conffiles/etc/pam.d/common-session.d/20system-setup-login-message
base/univention-system-setup/conffiles/usr/share/univention-management-console/modules/setup-certificate.xml
base/univention-updater/conffiles/etc/apt/apt.conf.d/55user_agent
base/univention-updater/conffiles/etc/apt/mirror.list
base/univention-updater/conffiles/etc/apt/sources.list.d/15_ucs-online-version.list
base/univention-updater/conffiles/etc/apt/sources.list.d/20_ucs-online-component.list
base/univention-updater/conffiles/etc/cron.d/univention-updater-check
base/univention-updater/conffiles/etc/logrotate.d/univention-updater
container/univention-docker-container-mode/conffiles/etc/apt/apt.conf.d/docker-clean
container/univention-docker/conffiles/etc/default/docker
container/univention-docker/conffiles/etc/docker/daemon.json
container/univention-docker/conffiles/etc/systemd/system/docker.service.d/http-proxy.conf
desktop/univention-kde/conffiles/etc/dbus-1/system.d/devices.conf
desktop/univention-kde/conffiles/usr/share/univention-kde-profiles/kde4-menu/.kde/share/config/kickoffrc
desktop/univention-kdm/conffiles/etc/default/kdm.d/ucsrc
desktop/univention-mozilla-firefox/conffiles/etc/firefox-esr/univention.cfg
desktop/univention-mozilla-firefox/conffiles/etc/firefox-esr/univention.js
desktop/univention-x-core/conffiles/etc/X11/xorg.conf
doc/developer-reference/ucr/hosts/conffiles/etc/hosts.allow.d/50dynamic
doc/developer-reference/ucr/hosts/conffiles/etc/hosts.deny.d/50dynamic
mail/univention-antivir-mail/conffiles/etc/amavis/conf.d/15-content_filter_mode
mail/univention-antivir-mail/conffiles/etc/amavis/conf.d/60-univention
mail/univention-antivir-mail/conffiles/etc/clamav/freshclam.conf
mail/univention-fetchmail/conffiles/etc/default/fetchmail
mail/univention-fetchmail/conffiles/etc/ldap/slapd.conf.d/58univention-fetchmail_acl-settings
mail/univention-mail-dovecot/conffiles/etc/default/dovecot
mail/univention-mail-dovecot/conffiles/etc/dovecot/conf.d/10-auth.conf
mail/univention-mail-dovecot/conffiles/etc/dovecot/conf.d/10-logging.conf
mail/univention-mail-dovecot/conffiles/etc/dovecot/conf.d/10-mail.conf
mail/univention-mail-dovecot/conffiles/etc/dovecot/conf.d/10-master.conf
mail/univention-mail-dovecot/conffiles/etc/dovecot/conf.d/10-ssl.conf
mail/univention-mail-dovecot/conffiles/etc/dovecot/conf.d/15-mailboxes.conf
mail/univention-mail-dovecot/conffiles/etc/dovecot/conf.d/20-imap.conf
mail/univention-mail-dovecot/conffiles/etc/dovecot/conf.d/20-managesieve.conf
mail/univention-mail-dovecot/conffiles/etc/dovecot/conf.d/20-pop3.conf
mail/univention-mail-dovecot/conffiles/etc/dovecot/conf.d/90-quota.conf
mail/univention-mail-dovecot/conffiles/etc/dovecot/conf.d/95-quota-status.conf
mail/univention-mail-dovecot/conffiles/etc/dovecot/dovecot-ldap.conf.ext
mail/univention-mail-dovecot/conffiles/etc/dovecot/dovecot.conf
mail/univention-mail-dovecot/conffiles/etc/logrotate.d/dovecot
mail/univention-mail-dovecot/conffiles/etc/pam.d/dovecot
mail/univention-mail-dovecot/conffiles/etc/rsyslog.d/dovecot.conf
mail/univention-mail-dovecot/conffiles/usr/share/dovecot/protocols.d/imapd.protocol
mail/univention-mail-dovecot/conffiles/usr/share/dovecot/protocols.d/pop3d.protocol
mail/univention-mail-dovecot/conffiles/var/lib/dovecot/sieve/default.sieve
mail/univention-mail-postfix/conffiles/etc/aliases
mail/univention-mail-postfix/conffiles/etc/cron.d/univention-mail-postfix
mail/univention-mail-postfix/conffiles/etc/listfilter.secret
mail/univention-mail-postfix/conffiles/etc/pam.d/smtp
mail/univention-mail-postfix/conffiles/etc/postfix/ldap.canonicalrecipient.d/10_base
mail/univention-mail-postfix/conffiles/etc/postfix/ldap.canonicalsender.d/10_base
mail/univention-mail-postfix/conffiles/etc/postfix/ldap.distlist.d/10_base
mail/univention-mail-postfix/conffiles/etc/postfix/ldap.external_aliases.d/10_base
mail/univention-mail-postfix/conffiles/etc/postfix/ldap.groups.d/10_base
mail/univention-mail-postfix/conffiles/etc/postfix/ldap.saslusermapping.d/10_base
mail/univention-mail-postfix/conffiles/etc/postfix/ldap.sharedfolderlocal.d/10_base
mail/univention-mail-postfix/conffiles/etc/postfix/ldap.sharedfolderlocal_aliases.d/10_base
mail/univention-mail-postfix/conffiles/etc/postfix/ldap.sharedfolderremote.d/10_base
mail/univention-mail-postfix/conffiles/etc/postfix/ldap.transport.d/10_base
mail/univention-mail-postfix/conffiles/etc/postfix/ldap.virtual.d/10_base
mail/univention-mail-postfix/conffiles/etc/postfix/ldap.virtual_mailbox.d/10_base
mail/univention-mail-postfix/conffiles/etc/postfix/ldap.virtualdomains.d/10_base
mail/univention-mail-postfix/conffiles/etc/postfix/ldap.virtualwithcanonical.d/10_base
mail/univention-mail-postfix/conffiles/etc/postfix/main.cf.d/10_general
mail/univention-mail-postfix/conffiles/etc/postfix/main.cf.d/30_maps
mail/univention-mail-postfix/conffiles/etc/postfix/main.cf.d/40_postscreen
mail/univention-mail-postfix/conffiles/etc/postfix/main.cf.d/50_restrictions
mail/univention-mail-postfix/conffiles/etc/postfix/main.cf.d/60_tls
mail/univention-mail-postfix/conffiles/etc/postfix/main.cf.d/80_delivery
mail/univention-mail-postfix/conffiles/etc/postfix/main.cf.d/99_local
mail/univention-mail-postfix/conffiles/etc/postfix/master.cf.d/10_services
mail/univention-mail-postfix/conffiles/etc/postfix/master.cf.d/30_antivir
mail/univention-mail-postfix/conffiles/etc/postfix/master.cf.d/70_policy
mail/univention-mail-postfix/conffiles/etc/postfix/master.cf.d/99_local
mail/univention-mail-postfix/conffiles/etc/postfix/tls_policy
mail/univention-mail-postfix/conffiles/etc/postfix/transport.d/10_base
mail/univention-postgrey/conffiles/etc/default/postgrey
mail/univention-spamassassin/conffiles/etc/cron.daily/univention-spamassassin
mail/univention-spamassassin/conffiles/etc/default/spamassassin
mail/univention-spamassassin/conffiles/etc/logrotate.d/univention-spamassassin
mail/univention-spamassassin/conffiles/etc/spamassassin/90_univention.cf
mail/univention-spamassassin/conffiles/usr/sbin/univention-sa-learn
management/univention-admingrp-user-passwordreset/conffiles/etc/ldap/slapd.conf.d/65admingrp-user-passwordreset
management/univention-appcenter/conffiles/etc/apache2/sites-available/000-default.d/10univention-appcenter
management/univention-appcenter/conffiles/etc/apache2/sites-available/ssl.d/10univention-appcenter
management/univention-appcenter/conffiles/etc/logrotate.d/univention-appcenter
management/univention-appcenter/conffiles/etc/postgresql/9.4/main/pg_hba.conf.d/10-appcenter
management/univention-appcenter/conffiles/etc/postgresql/9.6/main/pg_hba.conf.d/10-appcenter
management/univention-appcenter/conffiles/etc/univention/service.info/services/univention-appcenter.cfg
management/univention-appcenter/conffiles/usr/share/univention-management-console/i18n/de/apps.mo
management/univention-appcenter/conffiles/usr/share/univention-management-console/modules/apps.xml
management/univention-directory-listener/conffiles/etc/logrotate.d/univention-directory-listener
management/univention-directory-logger/conffiles/etc/ldap/slapd.conf.d/41univention-directory-logger
management/univention-directory-logger/conffiles/etc/logrotate.d/univention-directory-logger
management/univention-directory-manager-modules/conffiles/etc/logrotate.d/univention-directory-manager
management/univention-directory-manager-rest/conffiles/etc/apache2/sites-available/univention-udm.conf
management/univention-directory-notifier/conffiles/etc/logrotate.d/univention-directory-notifier
management/univention-directory-policy/conffiles/etc/cron.d/univention-directory-policy
management/univention-directory-policy/conffiles/etc/logrotate.d/univention-directory-policy
management/univention-directory-replication/conffiles/etc/logrotate.d/univention-directory-replication
management/univention-directory-reports/conffiles/etc/logrotate.d/univention-directory-reports
management/univention-directory-reports/conffiles/etc/univention/directory/reports/config.ini
management/univention-join/conffiles/etc/logrotate.d/univention-join
management/univention-ldap-overlay-memberof/conffiles/etc/ldap/slapd.conf.d/41univention-ldap-overlay-memberof
management/univention-ldap/base.ldif
management/univention-ldap/conffiles/etc/cron.d/univention-ldap
management/univention-ldap/conffiles/etc/cron.d/univention-ldap-server
management/univention-ldap/conffiles/etc/init.d/slapd
management/univention-ldap/conffiles/etc/ldap/ldap.conf
management/univention-ldap/conffiles/etc/ldap/slapd.conf.d/10univention-ldap-server_schema
management/univention-ldap/conffiles/etc/ldap/slapd.conf.d/25univention-ldap-server_local-schema
management/univention-ldap/conffiles/etc/ldap/slapd.conf.d/30univention-ldap-server_head
management/univention-ldap/conffiles/etc/ldap/slapd.conf.d/31modules
management/univention-ldap/conffiles/etc/ldap/slapd.conf.d/39monitor
management/univention-ldap/conffiles/etc/ldap/slapd.conf.d/40univention-ldap-server_database
management/univention-ldap/conffiles/etc/ldap/slapd.conf.d/60univention-ldap-server_acl-master
management/univention-ldap/conffiles/etc/ldap/slapd.conf.d/60univention-ldap-server_acl-slave
management/univention-ldap/conffiles/etc/ldap/slapd.conf.d/63univention-ldap-server_acl-master-password
management/univention-ldap/conffiles/etc/ldap/slapd.conf.d/64univention-ldap-server_acl-master-admin-settings
management/univention-ldap/conffiles/etc/ldap/slapd.conf.d/70univention-ldap-server_acl-master-end
management/univention-ldap/conffiles/etc/ldap/slapd.conf.d/70univention-ldap-server_acl-slave-end
management/univention-ldap/conffiles/etc/ldap/slapd.conf.d/99translog
management/univention-ldap/conffiles/etc/nsswitch.conf
management/univention-ldap/conffiles/var/lib/univention-ldap/ldap/DB_CONFIG
management/univention-ldap/conffiles/var/lib/univention-ldap/translog/DB_CONFIG
management/univention-management-console/conffiles/etc/apache2/sites-available/univention-proxy.conf
management/univention-management-console/conffiles/etc/apache2/sites-available/univention.conf
management/univention-management-console/conffiles/etc/ldap/sasl2/slapd.conf
management/univention-management-console/conffiles/etc/logrotate.d/univention-management-console
management/univention-management-console/conffiles/etc/pam.d/univention-management-console.d/30_tally
management/univention-management-console/conffiles/etc/pam.d/univention-management-console.d/40_saml
management/univention-management-console/conffiles/etc/pam.d/univention-management-console.d/50_base
management/univention-management-console/conffiles/etc/pam.d/univention-management-console.d/60_account
management/univention-management-console/conffiles/usr/share/univention-management-console/meta.json
management/univention-portal/conffiles/etc/apache2/sites-available/univention-portal.conf
management/univention-portal/conffiles/etc/logrotate.d/univention-portal
management/univention-portal/conffiles/usr/share/univention-portal/apps.json
management/univention-server-overview/conffiles/etc/apache2/sites-available/univention-server-overview.conf
management/univention-web/conffiles/usr/share/univention-web/js/umc/hooks.json
management/univention-web/conffiles/var/www/univention/languages.json
nagios/univention-nagios/conffiles/etc/cron.d/univention-nagios
nagios/univention-nagios/conffiles/etc/nagios/apache2.conf
nagios/univention-nagios/conffiles/etc/nagios/cgi.cfg
nagios/univention-nagios/conffiles/etc/nagios/nagios.cfg
nagios/univention-nagios/conffiles/etc/nagios/nrpe.cfg
nagios/univention-nagios/conffiles/etc/pam.d/nagios
nagios/univention-nagios/conffiles/etc/security/access-nagios.conf
packaging/ucslint/testframework/0004-1/conffiles/etc/lilo.conf
packaging/ucslint/testframework/0004-10/conffiles/etc/lilo.conf
packaging/ucslint/testframework/0004-11/conffiles/etc/lilo.conf
packaging/ucslint/testframework/0004-12/conffiles/etc/hosts.d/99-ucs-test
packaging/ucslint/testframework/0004-12/conffiles/etc/lilo.conf
packaging/ucslint/testframework/0004-13/conffiles/etc/hosts.d/99-ucs-test
packaging/ucslint/testframework/0004-13/conffiles/etc/lilo.conf
packaging/ucslint/testframework/0004-14-15/conffiles/etc/lilo.conf
packaging/ucslint/testframework/0004-16-17/conffiles/etc/lilo.conf
packaging/ucslint/testframework/0004-18/conffiles/etc/lilo.conf
packaging/ucslint/testframework/0004-19-20/conffiles/etc/lilo.conf
packaging/ucslint/testframework/0004-21-22/conffiles/etc/lilo.conf
packaging/ucslint/testframework/0004-23/conffiles/etc/lilo.conf
packaging/ucslint/testframework/0004-24/conffiles/etc/lilo.conf
packaging/ucslint/testframework/0004-25-26/conffiles/etc/lilo.conf
packaging/ucslint/testframework/0004-28/conffiles/etc/lilo.conf
packaging/ucslint/testframework/0004-29/conffiles/etc/hosts.d/99-ucs-test
packaging/ucslint/testframework/0004-29/conffiles/etc/lilo.conf
packaging/ucslint/testframework/0004-3/conffiles/etc/lilo.conf
packaging/ucslint/testframework/0004-30/conffiles/etc/lilo.conf
packaging/ucslint/testframework/0004-31/conffiles/foo
packaging/ucslint/testframework/0004-4-5/conffiles/etc/lilo.conf
packaging/ucslint/testframework/0004-6/conffiles/etc/lilo.conf
packaging/ucslint/testframework/0004-7-8/conffiles/etc/lilo.conf
packaging/ucslint/testframework/0004-9/conffiles/etc/lilo.conf
packaging/ucslint/testframework/0010-2-3-4-5/conffiles/etc/cron.daily/univention-ssl-validity
packaging/ucslint/testframework/0011-2-3-4-5-6-7-8/conffiles/etc/cron.daily/univention-ssl-validity
packaging/ucslint/testframework/0013-2/conffiles/etc/cron.daily/univention-ssl-validity
packaging/ucslint/testframework/0015-2/conffiles/etc/cron.daily/univention-ssl-validity
packaging/univention-directory-manager-module-example/conffiles/etc/ldap/slapd.conf.d/11univention-ip-phone_schema
packaging/univention-package-template/conffiles/etc/ucsversion
saml/univention-saml/conffiles/etc/apache2/sites-available/univention-saml.conf
saml/univention-saml/conffiles/etc/ldap/slapd.conf.d/11univention-saml-schema
saml/univention-saml/conffiles/etc/simplesamlphp/00authsources.php
saml/univention-saml/conffiles/etc/simplesamlphp/01authsources-negotiate.php
saml/univention-saml/conffiles/etc/simplesamlphp/config.php
saml/univention-saml/conffiles/etc/simplesamlphp/metadata/00_saml20-idp-hosted.php
saml/univention-saml/conffiles/etc/stunnel/univention_saml.conf
services/univention-ad-connector/conffiles/etc/ldap/slapd.conf.d/11univention-ad-connector-exchange_schema
services/univention-ad-connector/conffiles/etc/logrotate.d/univention-ad-connector
services/univention-ad-connector/conffiles/etc/univention/connector/ad/mapping
services/univention-admin-diary/conffiles/etc/logrotate.d/univention-admindiary
services/univention-admin-diary/conffiles/etc/rsyslog.d/univention-admin-diary-backend.conf
services/univention-admin-diary/conffiles/etc/rsyslog.d/univention-admin-diary-client.conf
services/univention-apache/conffiles/etc/apache2/conf-available/ucs.conf
services/univention-apache/conffiles/etc/apache2/mods-available/proxy.conf
services/univention-apache/conffiles/etc/apache2/mods-available/ssl.conf
services/univention-apache/conffiles/etc/apache2/sites-available/ssl.d/00start
services/univention-apache/conffiles/etc/apache2/sites-available/ssl.d/10hsts
services/univention-apache/conffiles/etc/apache2/sites-available/univention-vhosts.conf
services/univention-apache/conffiles/etc/apache2/ucs-sites.conf.d/ucs-sites.conf
services/univention-apache/conffiles/var/www/robots.txt
services/univention-bind/conffiles/etc/bind/named.conf
services/univention-bind/conffiles/etc/bind/named.conf.proxy
services/univention-bind/conffiles/etc/bind/named.conf.samba4
services/univention-bind/conffiles/etc/init.d/bind9
services/univention-bind/conffiles/etc/systemd/system/bind9.service.d/10-configure-backend.conf
services/univention-cloud-init/conffiles/etc/cloud/cloud.cfg.d/91-univention-cloud-init.cfg
services/univention-dansguardian/conffiles/etc/dansguardian/dansguardian.conf
services/univention-dansguardian/conffiles/etc/dansguardian/dansguardianf1.conf
services/univention-dansguardian/conffiles/etc/dansguardian/dansguardianfX.conf
services/univention-dansguardian/conffiles/etc/dansguardian/lists/authplugins/ipgroups
services/univention-dansguardian/conffiles/etc/dansguardian/lists/bannedextensionlist
services/univention-dansguardian/conffiles/etc/dansguardian/lists/bannediplist
services/univention-dansguardian/conffiles/etc/dansguardian/lists/bannedmimetypelist
services/univention-dansguardian/conffiles/etc/dansguardian/lists/bannedregexpurllist
services/univention-dansguardian/conffiles/etc/dansguardian/lists/bannedsitelist
services/univention-dansguardian/conffiles/etc/dansguardian/lists/bannedurllist
services/univention-dansguardian/conffiles/etc/dansguardian/lists/contentscanners/exceptionvirusextensionlist
services/univention-dansguardian/conffiles/etc/dansguardian/lists/contentscanners/exceptionvirusmimetypelist
services/univention-dansguardian/conffiles/etc/dansguardian/lists/contentscanners/exceptionvirussitelist
services/univention-dansguardian/conffiles/etc/dansguardian/lists/contentscanners/exceptionvirusurllist
services/univention-dansguardian/conffiles/etc/dansguardian/lists/exceptionextensionlist
services/univention-dansguardian/conffiles/etc/dansguardian/lists/exceptioniplist
services/univention-dansguardian/conffiles/etc/dansguardian/lists/exceptionmimetypelist
services/univention-dansguardian/conffiles/etc/dansguardian/lists/exceptionregexpurllist
services/univention-dansguardian/conffiles/etc/dansguardian/lists/exceptionsitelist
services/univention-dansguardian/conffiles/etc/dansguardian/lists/exceptionurllist
services/univention-dansguardian/conffiles/etc/dansguardian/lists/filtergroupslist
services/univention-dhcp/conffiles/etc/dhcp/dhcpd.conf
services/univention-dhcp/conffiles/etc/logrotate.d/univention-dhcp
services/univention-mariadb/conffiles/etc/mysql/mariadb.conf.d/60-ucr.cnf
services/univention-mariadb/conffiles/lib/systemd/system/mariadb.service.d/univention.conf
services/univention-nfs/conffiles/etc/default/nfs-common
services/univention-nfs/conffiles/etc/default/nfs-kernel-server
services/univention-nfs/conffiles/etc/default/quota
services/univention-nfs/conffiles/etc/modprobe.d/nfs-kernel-lockd.conf
services/univention-pkgdb/conffiles/etc/apt/apt.conf.d/61invoke
services/univention-pkgdb/conffiles/etc/logrotate.d/univention-pkgdb
services/univention-pkgdb/conffiles/etc/postgresql/9.4/main/pg_hba.conf.d/10-pg_hba.conf
services/univention-pkgdb/conffiles/etc/postgresql/9.6/main/pg_hba.conf.d/10-pg_hba.conf
services/univention-postgresql/conffiles/etc/postgresql/9.4/main/postgresql.conf
services/univention-postgresql/conffiles/etc/postgresql/9.6/main/pg_hba.conf.d/99-pg_hba.conf
services/univention-postgresql/conffiles/etc/postgresql/9.6/main/postgresql.conf
services/univention-postgresql/conffiles/etc/postgresql/pam_ldap.conf
services/univention-printquota/conffiles/etc/postgresql/9.4/main/pg_hba.conf.d/11-pg_hba.conf
services/univention-printquota/conffiles/etc/postgresql/9.6/main/pg_hba.conf.d/11-pg_hba.conf
services/univention-printquota/conffiles/etc/pykota/pykota.conf
services/univention-printquota/conffiles/etc/pykota/pykotadmin.conf
services/univention-printserver/conffiles/etc/cron.d/univention-printserver-pdf
services/univention-printserver/conffiles/etc/cups/cups-files.conf
services/univention-printserver/conffiles/etc/cups/cups-pdf.conf
services/univention-printserver/conffiles/etc/cups/cupsd.conf.d/01cupsd-base
services/univention-printserver/conffiles/etc/cups/cupsd.conf.d/02cups-access-limit
services/univention-printserver/conffiles/etc/cups/cupsd.conf.d/03cups-include-local
services/univention-printserver/conffiles/etc/logrotate.d/univention-printserver
services/univention-radius/conffiles/etc/freeradius/3.0/mods-available/eap
services/univention-radius/conffiles/etc/freeradius/3.0/mods-available/ldap
services/univention-radius/conffiles/etc/freeradius/3.0/mods-available/mschap
services/univention-radius/conffiles/etc/freeradius/3.0/proxy.conf
services/univention-radius/conffiles/etc/freeradius/3.0/radiusd.conf
services/univention-radius/conffiles/etc/freeradius/3.0/sites-available/default
services/univention-radius/conffiles/etc/freeradius/3.0/sites-available/inner-tunnel
services/univention-radius/usr/share/univention-radius/59univention-radius.acl
services/univention-radius/usr/share/univention-radius/66univention-radius.acl
services/univention-s4-connector/conffiles/etc/logrotate.d/univention-s4-connector
services/univention-samba/conffiles/etc/logrotate.d/samba
services/univention-samba/conffiles/etc/logrotate.d/univention-samba

kolab2/horde-webmailer/horde-webmail/imp/config/conf.php
<?php
/* CONFIG START. DO NOT CHANGE ANYTHING IN OR AFTER THIS LINE. */
// $Horde: imp/config/conf.xml,v 1.53.2.43 2009-07-02 06:18:15 slusarz Exp $
$conf['spell']['driver'] = 'aspell';
$conf['utils']['gnupg'] = '/usr/bin/gpg';
$conf['utils']['gnupg_keyserver'] = array('pgp.mit.edu');
$conf['utils']['gnupg_timeout'] = 10;
$conf['utils']['openssl_binary'] = '/kolab/bin/openssl';
$conf['menu']['apps'] = array();
$conf['user']['select_sentmail_folder'] = true;
$conf['user']['allow_resume_all_in_drafts'] = true;
$conf['user']['allow_folders'] = true;
$conf['user']['allow_resume_all'] = false;
$conf['user']['allow_view_source'] = true;
$conf['user']['alternate_login'] = false;
$conf['user']['redirect_on_logout'] = false;
$conf['user']['default_view'] = 'dimp';
$conf['user']['force_view'] = 'dimp';
$conf['user']['select_view'] = false;
$conf['server']['change_server'] = false;
$conf['server']['change_port'] = false;
$conf['server']['change_protocol'] = false;
$conf['server']['change_smtphost'] = false;
$conf['server']['change_smtpport'] = false;
$conf['server']['server_list'] = 'none';
$conf['server']['fixed_folders'] = array();
$conf['server']['sort_limit'] = 0;
$conf['server']['cache_folders'] = true;
$conf['server']['token_lifetime'] = 1800;
$conf['server']['cachejs'] = 'none';
$conf['server']['cachecss'] = 'none';
$conf['mailbox']['show_preview'] = false;
$conf['fetchmail']['show_account_colors'] = false;
$conf['fetchmail']['size_limit'] = 4000000;
$conf['msgcache']['compress']['method'] = 'gzip';
$conf['msgcache']['use_compress'] = true;
$conf['msgcache']['preview_size'] = 1000;
$conf['msgcache']['lifetime'] = 2592000;
$conf['msgcache']['use_msgcache'] = true;
$conf['mlistcache']['lifetime'] = 604800;
$conf['mlistcache']['use_mlistcache'] = true;
$conf['msgsettings']['filtering']['words'] = './config/filter.txt';
$conf['msgsettings']['filtering']['replacement'] = '****';
$conf['spam']['reporting'] = false;
$conf['notspam']['reporting'] = false;
$conf['print']['add_printedby'] = true;
$conf['msg']['prepend_header'] = true;
$conf['msg']['append_trailer'] = false;
$conf['compose']['allow_receipts'] = true;
$conf['compose']['special_characters'] = true;
$conf['compose']['use_vfs'] = true;
$conf['compose']['link_all_attachments'] = false;
$conf['compose']['link_attachments_notify'] = true;
$conf['compose']['link_attachments'] = true;
$conf['compose']['attach_size_limit'] = 0;
$conf['compose']['attach_count_limit'] = 0;
$conf['compose']['reply_limit'] = 200000;
$conf['hooks']['vinfo'] = false;
$conf['hooks']['postlogin'] = false;
$conf['hooks']['postsent'] = false;
$conf['hooks']['signature'] = false;
$conf['hooks']['trailer'] = false;
$conf['hooks']['fetchmail_filter'] = false;
$conf['hooks']['mbox_redirect'] = true;
$conf['hooks']['mbox_icon'] = true;
$conf['hooks']['spam_bounce'] = false;
$conf['hooks']['msglist_format'] = false;
$conf['hooks']['display_folder'] = true;
$conf['maillog']['use_maillog'] = true;
$conf['sentmail']['params']['threshold'] = 60;
$conf['sentmail']['params']['limit_period'] = 24;
$conf['sentmail']['params']['database'] = dirname(__FILE__) . '/../../../../../webclient_data/storage/imp.db';
$conf['sentmail']['params']['mode'] = '0644';
$conf['sentmail']['params']['charset'] = 'utf-8';
$conf['sentmail']['params']['phptype'] = 'sqlite';
$conf['sentmail']['params']['table'] = 'imp_sentmail';
$conf['sentmail']['params']['driverconfig'] = 'custom';
$conf['sentmail']['driver'] = 'sql';
$conf['tasklist']['use_tasklist'] = true;
$conf['notepad']['use_notepad'] = true;
/* CONFIG END. DO NOT CHANGE ANYTHING IN OR BEFORE THIS LINE. */